PersonApi/serverless.yml

```diff
@@ -115,6 +115,45 @@ resources:
                           - Ref: 'AWS::Region'
                           - Ref: 'AWS::AccountId'
                           - 'table/Persons'
+  ## Steps required for AppConfig cloudformation
+    AppConfig:
+      Type: AWS::AppConfig::Application
+      Properties:
+        Name: $housing-tl/${self:provider.stage}/person-api"
+    DevEnvironment:
+      Type: AWS::AppConfig::Environment
+      Properties:
+        ApplicationId: !Ref EnvironmentApplication
+        Name: "Development"
+    ConfigurationProfile:
+      Type: AWS::AppConfig::ConfigurationProfile
+      Properties:
+        ApplicationId: !Ref ConfigurationApplication
+        Name: "PersonConfigurationProfile"
+        LocationUri: ${ssm:/person-api/${self:provider.stage}/arn}
+        Validators:
```

Comment:
I think this should be a `version` with `JSON` content, rather than a validator?